README.md

```diff
@@ -1184,7 +1184,7 @@ console.log(weakset.has(b));   // => false
 * Weak-collections polyfill stores values as hidden properties of keys. It works correct and not leak in most cases. However, it is desirable to store a collection longer than its keys.
 
 #### ECMAScript: Typed Arrays
-Implementations and fixes `ArrayBuffer`, `DataView`, typed arrays constructors, static and prototype methods. Typed Arrays work only in environments with support descriptors (IE9+), `ArrayBuffer` and `DataView` should work anywhere.
+Implementations and fixes of `ArrayBuffer`, `DataView`, typed arrays constructors, static and prototype methods. Typed arrays work only in environments with support descriptors (IE9+), `ArrayBuffer` and `DataView` should work anywhere.
```

Comment:
s/of/for

Also, “Typed Arrays” should retain their capitalization, since they’re not really a form of arrays, they’re their own primitive.

Reply:
Thanks -)

Reply:
* to clarify, “implementations of and fixes for” is probably more correct, but if there’s only one of them i think the latter sounds better.